docs/DeveloperGuide.md

```diff
@@ -923,6 +923,23 @@ testers are expected to do more *exploratory* testing.
 
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
+1. _{ more test cases …​ }_
```

Comment:
Can remove this more test cases line, it was supposed to be for us to add stuff.